acceptance-tests/features/steps/make_deed_effective_definitions.rb

```diff
@@ -0,0 +1,35 @@
+Given(/^I have made the created deed effective$/) do
+  sign_the_deed(@deed_id, 'SIGNATURE', '1')
+  make_deed_effective(@deed_id)
+end
+
+When(/^I get the deed from the api$/) do
+  @deed = get_deed_data(@deed_id)
+end
+
+Then(/^the signature is applied to the deed$/) do
+  assert(@deed['deed']['operative-deed']['registrars-signature'],
+         'Signature not applied')
+end
+
+Then(/^the date is applied to the deed$/) do
+  assert(@deed['deed']['operative-deed']['date-effective'],
+         'Date not applied')
+end
```

Comment:
My only worry here is that we are checking that something exists within the date-effective part of the deed json but not what is in there. The acceptance criteria states that it should be a timestamp (current date time) is there a way we could expand this to check not only is something there that it is of type datetime and possibly that it contains the correct date?